src/main/java/com/hyperwallet/clientsdk/util/Multipart.java

```diff
@@ -0,0 +1,59 @@
+package com.hyperwallet.clientsdk.util;
+
+import com.hyperwallet.clientsdk.model.HyperwalletVerificationDocument;
+import net.minidev.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.HashMap;
+
+public class Multipart {
+
+    List<MultipartData> multipartList;
+
+    public List<MultipartData> getMultipartList() {
+        return multipartList;
+    }
+
+    public void setMultipartList(List<MultipartData> multipartList) {
+        this.multipartList = multipartList;
+    }
+
+    public void add(MultipartData multipartData){
+        if (multipartList == null) {
+            multipartList = new ArrayList<MultipartData>();
+        }
+        multipartList.add(multipartData);
+    }
+
+    public static class MultipartData {
+        private String contentType; //json, img
+        private Map<String, String> entity; //name, content
+        private String contentDisposition;
```

Comment:
If all these attributes are required for the MultipartData creation then you should have a constructor with all these parameters, make them final, and remove set methods.

Reply:
You forgot to add final on them